src/main/java/com/kevin/tiertagger/TierTagger.java

```diff
@@ -32,7 +38,7 @@ public void onInitialize() {
                     JsonObject o = new Gson().fromJson(s, JsonObject.class);
                     o.entrySet().forEach(e -> tiers.put(e.getKey(), e.getValue().getAsString()));
                 })
-                .whenComplete((s, t) -> System.out.println(tiers.get("Ooh_Netiyiy") + " | Ooh_Netiyiy"));
+                .whenComplete((s, t) -> System.out.println("success"));
```

Comment:
unneeded print, you should probably rather print something when there is an error (or rather, log it with slf4j)

Reply:
I just changed what netiyiy wrote but ok